.gitignore

```diff
@@ -14,3 +14,6 @@
 # Project-local glide cache, RE: https://github.com/Masterminds/glide/issues/736
 .glide/
 .DS_Store
+
+# testing files
```

Comment:
Let's update this to the latest .gitignore provided by default for Golang related projects on GitHub. Also, entries like .glide/ can be removed. What do you think @mohsin-plivo?

Reply:
Updated